.github/workflows/check-openapi-change.yaml

```diff
@@ -0,0 +1,70 @@
+# This workflow will fetch the new openapi from the provided branch of https://github.com/Hyperfoil/Horreum and based on
+# that it will re-generate the Horreum raw client and check the build/tests are still working.
+# It could be tested running `gh act workflow_dispatch -e ./test/gha_workflows/workflow_dispatch_event_example.json`.
+name: Update Horreum auto-generated client
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: Branch or tag of https://github.com/Hyperfoil/Horreum
+        required: true
+  # this event should be triggered by Horreum repo using peter-evans/repository-dispatch@v2
+  repository_dispatch:
+    types: [ detected-horreum-openapi-change ]
+
+jobs:
+  check-openapi-change:
+    # TODO: find a way to set name including the branch we are checking
```

Comment:
>Which "name" are you trying to set?

I mean exactly the job name, such that you can easily inspect the check target from the job name rather than having to open the job and look for the branch.

>  name: check-openapi-change-${{ github.event.inputs.branch }}

This was a possible solution but as you highlighted I don't think this would work for `repository_dispatch` events, so maybe I can do the opposit and use `name: check-openapi-change-${{ github.event.client_payload.branch }}` which will be empty when triggered using `workflow_dispatch` which is fair as you had to do it manually.

Reply:
yeah maybe better this way